index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css" />
+    <title>Portfolio Project</title>
+</head>
+<body>
+
+    <div class="template-1">
+
+        <header>
+            <!-- Toolbar Nav -->
+    <nav class="toolbar-container">
+        <!-- Toolbar Name -->
+        <div class="toolbar-name-container">
+            <p class="toolbar-name">Wahab</p>
+        </div>
+        <!-- Toolbar Menu -->
+        <div class="toolbar-menu-container">
+            <img src="assets/Menu.svg" alt="Menu icon">
+        </div>
+    </nav>
```

Comment:
- [ OPTIONAL] Please make it a habit to indent your code well. Well-indented code is easy to read. You can use tool like prettier to auto-format your code on save

Reply:
ok!